src/static/css/options.css

```diff
@@ -11,7 +11,7 @@ body.version-lt-57 {
 }
 body[data-browser=Chrome] {
   padding: 0 1em 1em 1em;
-  min-width: 520px;
+  min-width: auto;
```

Comment:
It might be better to change it to:
```css
width: 520px;
max-width: 100%;
```

Reply:
It seems that both `min-width: auto;` and `max-width: 100%;` can be omitted, and still achieve the same results in Chrome and Edge..

Reply:
Without `max-width: 100%`, the option page has a fixed width, resulting in horizontal scrollbar in Edge.

Reply:
> 

But the following styles look like they work for me in Edge, without horizontal scrollbar.
`
body[data-browser=Chrome] {
  padding: 0 1em 1em 1em;
}
`